CITATION.cff

```diff
@@ -0,0 +1,20 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "de Sá"
+  given-names: "Fernando Henrique"
+ 
+- family-names: "Resende"
+  given-names: "Ximenes"
+
```

Comment:
@xresende  May I have to include Afonso here?

Reply:
@jquentino, I think the natural order here would be to put @xresende as first author, @afonsoharuo as second, @LuanaVilela  as third and only then myself and @murilobalves.  Even though I have more commits and lines edited, they started the project and came up with the structures we use today.

Reply:
Yes, I agree with @fernandohds564.

Reply:
Yes, it is fair, I did the change.

Reply:
thanks!